benchmarks/gogo/cpp_no_group/cpp_benchmark.cc

```diff
@@ -0,0 +1,253 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Why can't we reuse the old code?

Reply:
Done by add rule generate_cpp_no_group_benchmark_code